images/tier1/angry.svg

```diff
@@ -0,0 +1,3 @@
+<svg xmlns="http://www.w3.org/2000/svg" height="18" width="18" viewBox="0 0 18 18">
```

Comment:
Is this part of some kind of emoji collection? If yes, wondering if we should group them all under a separate category instead of `tier1`.